_gcode/G028.md

```diff
@@ -7,6 +7,7 @@ author: thinkyhead
 group: calibration
 
 codes: [ G28 ]
+related: [ G34 ]
```

Comment:
Lots of features will home if needed, so I’m not sure adding `G34` here is needed.

Reply:
ok